### ISSUE TITLE
New installer causes BSOD

### ISSUE THREAD

**Issue by hairagon** [Externoak/A32NX-installer#10]
After installing the v0.7 this morning my PC (windows 10 / 2004) BSODs when attempting to update Stable version.
After a full restart / reboot the application hangs. No other software open/running in the foreground.
Only event in the log around that time is "Failed with 0x57 retrieving AppModel Runtime status for package Windows." No mention of the app however.
I have recreated the target directory but no change in issue.
Process:
1) open app from desktop
2) select [STABLE] button.

App hangs and indicates non responsive. Did not get BSOD on this or occasions other than the first time I tried.
Windows 10 up to date (v2004),

**Comment by Externoak** [comment]
Thanks for the feedback.

Regarding the BSOD if it only happened once and it's not reproducible there is not much where to debug. Although the error code mention above seems to be related to your graphics card drivers.

"App hangs and indicates non responsive." What stage of the app does this happen and what download  server are you choosing?

**Comment by hairagon** [comment]
I have sorted at after odd behaviour on another program. My firewall (Bullguard) has always asked me to Allow or deny access when it encontered new software. On this occasion and a later one it did not. I switched the FWall off and the app ran perfectly as did the other troublesome one. When the AV/FW software was open it then asked for me to allow the downloader. 
From then on all was well.
Have since had a new vession of Bullguard, perhaps they knew of the issue.
Thanks for your time anyway,
Stay Safe.
